docker-compose.yml

```diff
@@ -11,6 +11,10 @@ services:
     - DESEC_NS_CARBONSERVER
     - DESEC_NS_CARBONOURNAME
     - DESEC_NS_APIKEY
+    - DESEC_NS_NAME
+    - DESEC_NS_SIGNALING_DOMAIN_ZONE_KEY_B64
```

Comment:
Variable name should include "private" so people know it's sensitive

Reply:
done